packages/messaging/test/iid-model-delete-token.test.ts

```diff
@@ -42,37 +42,38 @@ describe('Firebase Messaging > IIDModel.deleteToken()', function() {
 
   it('should delete on valid request', async function() {
     globalIIDModel = new IIDModel();
-    let stubbedFetch = sinon.stub(window, 'fetch');
-    stubbedFetch.returns(fetchMock.jsonOk(''));
+    sandbox.stub(window, 'fetch').returns(fetchMock.jsonOk('{}'));
     await globalIIDModel.deleteToken(fcmSenderId, fcmToken, fcmPushSet);
-    stubbedFetch.restore();
   });
 
   it('should handle fetch errors', async function() {
     globalIIDModel = new IIDModel();
     const errorMsg = 'invalid token';
-    let stubbedFetch = sinon.stub(window, 'fetch');
-    stubbedFetch.returns(fetchMock.jsonError(400, errorMsg));
+    const payload = JSON.stringify({
+      error: {
+        message: errorMsg
+      }
+    });
+
+    sandbox.stub(window, 'fetch').returns(fetchMock.jsonError(400, payload));
```

Comment:
fetchMock.jsonError already puts the error message into a JSON, you don't need the payload: https://github.com/firebase/firebase-js-sdk/blob/master/packages/messaging/test/testing-utils/mock-fetch.ts